test/e2e/specs/wp-inline-help-support-search-spec.js

```diff
@@ -32,11 +32,16 @@ before( async function () {
 describe( `[${ host }] Inline Help: (${ screenSize }) @parallel`, async function () {
 	this.timeout( mochaTimeOut );
 
-	step( 'Login and select a non My Home page', async function () {
+	step( 'Login and select a page that is not the My Home page', async function () {
 		const loginFlow = new LoginFlow( driver );
 
 		await loginFlow.loginAndSelectMySite();
 
+		// The "/home" route is the only one where the "FAB" inline help
+		// is not shown.
+		const sidebarComponent = await SidebarComponent.Expect( driver );
+		await sidebarComponent.selectSettings();
+
```

Comment:
https://github.com/Automattic/wp-calypso/blob/3601f2f8931f1de598e0740548d3029017cdfa67/test/e2e/lib/flows/login-flow.js#L244-L249

`loginAndSelectSettings` is in `loginFlow` already and matches this code. Let's use that instead.

Reply:
Pushed.